scripts/install.sh

```
@@ -438,24 +438,54 @@ fi
+    case "$OS_TYPE" in
+    "ubuntu" | "debian" | "raspbian")
+        apt-get update
+        apt-get install -y ca-certificates curl
+        install -m 0755 -d /etc/apt/keyrings
+        curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+        chmod a+r /etc/apt/keyrings/docker.asc
+
+        # Add the repository to Apt sources
+        echo \
+            "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+                  $(. /etc/os-release && echo "${UBUNTU_CODENAME:-$VERSION_CODENAME}") stable" |
```

Shellcheck (0.10.0)

[info] Not following: /etc/os-release was not specified as input (see shellcheck -x).

(SC1091)